lib/success.js

```diff
@@ -250,3 +291,24 @@ export default async function success(pluginConfig, context, { Octokit }) {
     throw new AggregateError(errors);
   }
 }
+
+export function loadAssociatedPRsQuery (shas) {
```

Comment:
there is no loading happening here, I'd call it something like 

```suggestion
export function buildAssociatedPRsQuery (shas) {
```